tasks/start.yml

```diff
@@ -3,5 +3,5 @@
   service: name=influxdb state=started enabled=yes
 
 - name: wait for influxdb port
-  wait_for: port={{influxdb_client_port}} host={{influxdb_bind_address | default('127.0.0.1') }} state=started timeout=30
+  wait_for: port={{influxdb_client_port}} host={{influxdb_bind_address | default('127.0.0.1') }} state=started timeout=180
```

Comment:
This is a rather large timeout but is that way because it has been necessary for some of the slower clusters we have installed to.